### PR TITLE
Update DisableWorkersInProject endpoint

### PR DESCRIPTION
**Pull request by KostinP**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/102?issue_count=37&issue_position=5&next_issue_id=100&prev_issue_id=103

### CODE REVIEW

File: src/ProjectService.Business/Commands/DisableWorkersInProjectCommand.cs [hunk 1]
```diff
@@ -1,30 +1,36 @@
...
             [FromServices] IProjectRepository repository,
-            [FromServices] IValidator<ProjectExpandedRequest> validator)
+            [FromServices] IAccessValidator accessValidator)
         {
             this.repository = repository;
-            this.validator = validator;
+            this.accessValidator = accessValidator;
         }
 
-        public void Execute(ProjectExpandedRequest request)
```
Comment (djino97) [inline src/ProjectService.Business/Commands/DisableWorkersInProjectCommand.cs:23]: Maybe use model ProjectExpandedRequest for this endpoint?

File: src/ProjectService.Business/Commands/DisableWorkersInProjectCommand.cs [hunk 2]
```diff
@@ -1,30 +1,36 @@
...
+        public void Execute(Guid projectId, IEnumerable<Guid> userIds)
         {
-            validator.ValidateAndThrowCustom(request);
```
Comment (djino97) [inline src/ProjectService.Business/Commands/DisableWorkersInProjectCommand.cs:25]: And need use validator for validation `IEnumerable<Guid>`

File: src/ProjectService.Data/ProjectRepository.cs [hunk 3]
```diff
@@ -70,26 +68,26 @@ public Guid EditProjectById(DbProject dbProject)
...
+        public void DisableWorkersInProject(Guid projectId, IEnumerable<Guid> userIds)
         {
             DbProject dbProject = provider.Projects
-                .FirstOrDefault(p => p.Id == request.Project.Id);
+                .FirstOrDefault(p => p.Id == projectId);
 
             if (dbProject == null)
             {
-                throw new NullReferenceException("Project with this Id does not exist.");
+                throw new NotFoundException($"Project with Id {projectId} does not exist.");
```
Comment (djino97) [inline src/ProjectService.Data/ProjectRepository.cs:78]: Maybe use BadRequestException from Kernel?

File: src/ProjectService.Business/Commands/DisableWorkersInProjectCommand.cs (hunk 1 above)
Comment (KostinP) [reply src/ProjectService.Business/Commands/DisableWorkersInProjectCommand.cs:23]: No.

File: src/ProjectService.Business/Commands/DisableWorkersInProjectCommand.cs (hunk 2 above)
Comment (KostinP) [reply src/ProjectService.Business/Commands/DisableWorkersInProjectCommand.cs:25]: Why?

Comment (KostinP) [reply src/ProjectService.Business/Commands/DisableWorkersInProjectCommand.cs:25]: Fixed.

**Review by djino97 — APPROVED** [review]

**Pull request merged**